src/AnalyzerConfiguration/NI.CSharp.Analyzers.targets

```diff
@@ -36,17 +36,12 @@
 
   <ItemGroup Condition="'$(NI_EnableAnalyzers)' == 'True'">
     <!-- Microsoft's SA analyzers -->
-    <Analyzer Include="$(PkgStyleCop_Analyzers)\analyzers\dotnet\cs\StyleCop.Analyzers.dll"/>
-    <Analyzer Include="$(PkgStyleCop_Analyzers)\analyzers\dotnet\cs\StyleCop.Analyzers.CodeFixes.dll"/>
+    <Analyzer Include="$(PkgStyleCop_Analyzers_Unstable)\analyzers\dotnet\cs\StyleCop.Analyzers.dll"/>
```

Comment:
NOTE: we're depending on the beta version of StyleCop to get its latest features.  

The beta version of StyleCop.Analyzers (1.2.0-beta.435) is a thin wrapper of StyleCop.Analyzers.Unstable (1.2.0.435) , 
it does not have the `analyzers` folder. We need to get the analyzers from StyleCop.Analyzers.Unstable .